torch/csrc/Module.cpp

```diff
@@ -1704,6 +1705,30 @@ Call this whenever a new thread is created in order to propagate values from
         return torch::should_allow_numbers_as_tensors(name);
       });
 
+  // FIXME(crcrpar): Better have `at::ScalarType` get mapped to `torch.dtype`
+  // Currently I see the second item of the key is displayed as
+  // e.g. `torch._C._te.ScalarType at 0x7fcf318adab0`
```

Comment:
Here's my question: How can I let `at::ScalarType` to be correctly cast to `torch.dtype` in Python?
Currently, this function emits (at::Device, std::string) and _foreach_utils maps the string to `torch.dtype`